packages/flutter/lib/src/rendering/tweens.dart

```diff
@@ -22,12 +21,12 @@ class FractionalOffsetTween extends Tween<FractionalOffset> {
   ///
```

Comment:
shouldn't Tween be `<T extends Object>` and have its `lerp` support returning null?